packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -256,19 +260,19 @@ class _CreationParams {
 }
 
 class _WebSettings {
-  _WebSettings({
-    this.javascriptMode,
-  });
-
+  _WebSettings({this.javascriptMode, this.zoom});
```

Comment:
The way this worked has changed a bit, please rebase and use the widget reference.